bin/run.sh

```diff
@@ -15,8 +15,11 @@ fi
 
 # Only process branches listed in DEPLOY_BRANCHES
 BRANCHES_TO_DEPLOY=($DEPLOY_BRANCHES)
-if [[ ! " ${BRANCHES_TO_DEPLOY[@]} " =~ " ${TRAVIS_BRANCH} " ]]; then
+if [[ ! " ${BRANCHES_TO_DEPLOY} " =~ " ${TRAVIS_BRANCH} " ]]; then
   # whatever you want to do when arr contains value
```

Comment:
It's not an array anymore, we should update this comment

Reply:
Will do.